woocommerce/wc-admin
# Added skip profiler functionality

Fixes #4568-partially

This PR adds the functionality to skip the onboarding profiler. Also, it leaves the `Store details` as incomplete in the `Home` screen.

### Accessibility




### Screenshots
![skip_profiler1](https://user-images.githubusercontent.com/1314156/86178149-3dd43b80-bafe-11ea-88f8-9918fafa177f.png)
![skip_profiler2](https://user-images.githubusercontent.com/1314156/86178157-40cf2c00-bafe-11ea-87c8-df552ce9dff7.png)

### Detailed test instructions:

To test this functionality I created the branch `add/4568_test` where I added a link to be able to skip the onboarding profiler.
1. Delete the profiler information with these two SQL sentences:
````
UPDATE `wp_options` SET `option_value` = 'a:0:{}' WHERE `wp_options`.`option_name` = 'woocommerce_onboarding_profile';
UPDATE `wp_options` SET `option_value` = 'no' WHERE `wp_options`.`option_name` = 'woocommerce_task_list_complete';
````
2. Activate the profiler setup wizard (going to `/wp-admin/admin.php?page=wc-settings` -> Setup wizard -> Profile setup wizard -> `Enable`).
3. Fill out a couple of onboarding steps.
4. Press `Skip`.
![skip_profiler3](https://user-images.githubusercontent.com/1314156/86178785-509b4000-baff-11ea-8a12-e9f1e70269ff.png)
5. Verify the task list on the `Home` screen has the `Store details` marked as incomplete.
6. Press `Store details`. The profiler should open again.
7. After pressing `Skip`, the profile setup wizard (step 2) should appear as disabled. Verify it's disabled when the profiler was skipped and enabled when it wasn't.

/cc @samueljseay This PR is related to the issue [4566](https://github.com/woocommerce/woocommerce-admin/issues/4566). I know Sam is working on it.



### Changelog Note:
Dev: Added skip profiler functionality
